test/acceptance/public-api-test.js

```diff
@@ -32,22 +33,25 @@ describe('public api', function () {
         '.template-lintrc.js': "throw Error('error happening during config loading');\n",
       });
 
-      expect(() => {
-        new Linter({
```

Comment:
Config loading no longer happens in constructor so changed to call `loadConfig` in this test.